src/js/utils/specs/ajax-helpers-spec.js

```diff
@@ -34,7 +34,7 @@ define(
                     $.ajax(testUrl);
 
                     expect(requests instanceof Array).toBeTruthy();
-                    expect(requests.hasOwnProperty('currentIndex')).toBeTruthy();
+                    expect(Object.prototype.hasOwnProperty.call(requests, 'currentIndex')).toBeTruthy();
```

Comment:
Agreed this is gross. `'prop` in object` works here afaik and is much prettier :+1: